1_fetch/cfg/stream_params.yml

```diff
@@ -1,4 +1,4 @@
 flow:
-  - '00060'
+  - '00095'
 stage:
-  - '00065'
+  - '00095'
```

Comment:
You might want to throw in some comments about changing the params here since they are no longer flow and stage codes. I know it is just because you are adapting the code quickly, but you should put a comment explicitly saying that.